ValveKeyValue/ValveKeyValue/Deserialization/KVAppendingObjectBuilder.cs

```diff
@@ -15,8 +15,26 @@ protected override void FinalizeState()
         {
             base.FinalizeState();
 
+			if (StateStack.Count <= 0)
+			{
+				// This will occur if an #include file does not exist.
+				return;
+			}
+
             var stateEntry = StateStack.Peek();
-            var originalStateEntry = originalBuilder.StateStack.Peek();
+            KVPartialState originalStateEntry;
+			if (originalBuilder.StateStack.Count <= 0)
+			{
+				// This will occur if a file consists only of #base or #include directives.
```

Comment:
Can we land up with an entirely empty file if it consists of only directives to load files that don't exist? How do/should we handle that?

Reply:
Yes, it is possible to end up with an entirely empty file. Currently, this causes the following error:

```
ValveKeyValue.KeyValueException : Builder is not in a fully completed state (stack count is 0).
```

I'm not sure what the best way to handle this would be. I'm pretty sure it's an unrecoverable error. The only thing I can think to do is make the error more human-friendly?